src/compile/data/impute.ts

```diff
@@ -0,0 +1,32 @@
+import {ImputeTransform} from '../../transform';
+import {duplicate} from '../../util';
+import {VgImputeTransform} from '../../vega.schema';
+import {DataFlowNode} from './dataflow';
+
+export class ImputeTransformNode extends DataFlowNode {
+  public clone() {
+    return new ImputeTransformNode(this.parent, duplicate(this.transform));
+  }
+
+  public producedFields() {
+    // typescript detects true as boolean type
+    return {[this.transform.impute]: true as true} ;
```

Comment:
This casting is weird.  What's wrong with boolean type?

Reply:
Getting this error 
`Type '{ [x: string]: boolean; }' is not assignable to type 'Dict<true>'.`

Reply:
Ok then.